src/main/java/next/controller/ApiUserController.java

```diff
@@ -0,0 +1,65 @@
+package next.controller;
+
+import core.annotation.web.Controller;
+import core.annotation.web.RequestMapping;
+import core.annotation.web.RequestMethod;
+import core.annotation.web.RequestParam;
+import core.db.DataBase;
+import core.mvc.JsonView;
+import core.mvc.ModelAndView;
+import next.dto.UserCreatedDto;
+import next.dto.UserUpdatedDto;
+import next.model.User;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+import org.springframework.web.bind.annotation.RequestBody;
+
+import javax.servlet.http.HttpServletResponse;
+
+import static core.mvc.tobe.HttpHeader.LOCATION;
+import static core.mvc.tobe.HttpStatus.*;
+
+@Controller
+public class ApiUserController {
+    private static final Logger logger = LoggerFactory.getLogger(ApiUserController.class);
+    private static final String BASE_URL_USER_API = "/api/users";
+
+    @RequestMapping(value = BASE_URL_USER_API, method = RequestMethod.POST)
+    public ModelAndView create(UserCreatedDto userCreatedDto, HttpServletResponse response) {
+        User user = userCreatedDto.toEntity();
```

Comment:
HTTP 본문을 해석해 주는 `ArgumentResolver` 의 구현체가 없어서 빈 객체가 생성 되고 있네요 😂 
리졸버를 구현하여 추가하거나 직접 `HttpServletRequest` 로 부터 읽어야 할 것 같아요!


```suggestion
    public ModelAndView create(HttpServletRequest request, HttpServletResponse response) {
        UserCreatedDto userCreatedDto = objectMapper.readValue(request.getInputStream(), UserCreatedDto.class);
        User user = userCreatedDto.toEntity();
```